iac/terraform/aws/provider.tf

```diff
@@ -0,0 +1,16 @@
+terraform {
```

Comment:
this should contain terraform version too

Reply:
We can not mention terraform version in provider or anywhere in terraform code. But of course we can hardcode provider version.